### PR TITLE
openexr & ilmbase 2.5.1

### PR DESCRIPTION
**Pull request by EricFromCanada**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
```
Error: 20 failed steps!
brew linkage --test rawtoaces
brew test --verbose rawtoaces
brew test --verbose caffe
brew linkage --test ctl
brew test --verbose ctl
brew linkage --test imagemagick
brew test --verbose imgproxy
brew linkage --test opencv
brew test --verbose opencv
brew linkage --test opencv@2
brew test --verbose opencv@2
brew linkage --test opencv@3
brew test --verbose opencv@3
brew linkage --test openimageio
brew test --verbose openimageio
brew linkage --test openvdb
brew test --verbose openvdb
brew linkage --test synfig
brew linkage --test vips
brew test --verbose vips
```

**Comment by tommyang** [comment]
2.5.1 is out https://github.com/AcademySoftwareFoundation/openexr/releases/tag/v2.5.1

File: Formula/opencv.rb [hunk 1]
```diff
@@ -91,10 +92,12 @@ def install
 
     mkdir "build" do
       system "cmake", "..", *args
+      inreplace "modules/core/version_string.inc", "#{HOMEBREW_SHIMS_PATH}/mac/super/", ""
```
Comment (MikeMcQuaid) [inline Formula/opencv.rb:95]: Does this still use shims at build-time?

File: Formula/opencv@2.rb [hunk 2]
```diff
@@ -66,6 +66,7 @@ def install
 
     mkdir "build" do
       system "cmake", "..", *args
+      inreplace "modules/core/version_string.inc", "#{HOMEBREW_SHIMS_PATH}/mac/super/", ""
```
Comment (MikeMcQuaid) [inline Formula/opencv@2.rb:69]: Does this still use shims at build-time?

File: Formula/opencv@3.rb [hunk 3]
```diff
@@ -77,10 +78,12 @@ def install
 
     mkdir "build" do
       system "cmake", "..", *args
+      inreplace "modules/core/version_string.inc", "#{HOMEBREW_SHIMS_PATH}/mac/super/", ""
```
Comment (MikeMcQuaid) [inline Formula/opencv@3.rb:81]: Does this still use shims at build-time?

File: Formula/opencv.rb (hunk 1 above)
Comment (EricFromCanada) [reply Formula/opencv.rb:95]: The part of this file being patched is only part of a build info screen, and not any parsed variables used in the build process.

File: Formula/opencv@2.rb (hunk 2 above)
Comment (EricFromCanada) [reply Formula/opencv@2.rb:69]: Same ⬆️

File: Formula/opencv@3.rb (hunk 3 above)
Comment (EricFromCanada) [reply Formula/opencv@3.rb:81]: Same ⬆️

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by Bo98** [comment]
Might be nice at some point to have something cross-platform so the `inreplaces` work on Linux (`Superenv.bin` exists but is marked as private).

**Pull request closed without merging**